kentikapi/internal/api_payloads/cloud_export.go

```diff
@@ -43,12 +45,18 @@ func cloudExportsFromPayload(exports []*cloudexportpb.CloudExport) ([]models.Clo
 // CloudExportFromPayload converts cloud export payload to model.
 func CloudExportFromPayload(ce *cloudexportpb.CloudExport) (*models.CloudExport, error) {
 	if ce == nil {
-		return nil, fmt.Errorf("cloud export response payload is nil")
+		return nil, kentikErrors.KentikError{
+			Codes: []kentikErrors.Code{kentikErrors.NotFound},
```

Comment:
```suggestion
			Codes: []kentikErrors.Code{kentikErrors.InvalidResponse},
```
This comes from invalid response format, so I suggest adding such status (only status; no check function for now).

Reply:
Done